io/include/pcl/io/hdl_grabber.h

```diff
@@ -189,6 +201,7 @@ namespace pcl
       static const int HDL_LASER_PER_FIRING = 32;
       static const int HDL_MAX_NUM_LASERS = 64;
       static const int HDL_FIRING_PER_PKT = 12;
+      pcl::RGB laser_rgb_mapping_[HDL_MAX_NUM_LASERS];
```

Comment:
Even though you're moving this to protected scope, which is correct, remember that VLPGrabber has its own laser_rgb_mapping_ member, which overshadows this one.

Reply:
Fixed